### PR TITLE
Add open source update article

### PR DESCRIPTION
**Pull request by ielerol**

### CODE REVIEW

File: _posts/2017-03-27-open source update.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
+---
```
Comment (marteki) [inline _posts/2017-03-27-open source update.md]: We should probably change the filename to be published on a different date, as it won't be published on March 27, 2017.

File: _posts/2017-03-27-open source update.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+published: true
+layout: post
+author: Catherine Farman
+title: "History and future of open source at CFPB"
+permalink: /articles/open-source-update/
+---
+
```
Comment (marteki) [inline _posts/2017-03-27-open source update.md]: Potentially out of scope for this PR: the front page of the cfpb.github.io site looks very odd with this PR included. Issues with spacing, and alignment with the top of the content versus the top of the sidebar.

screen shot 2017-05-22 at 3 48 53 pm

File: _posts/2017-03-27-open source update.md [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+including security risks or when the license is restricted.
+
+With an open source by default policy in place, 
+we’ve been able to collaborate with other government agencies 
+and the public at hackathon events where designers and developers come together 
+to work on open source software:
+
+- CFPB staff have been present at National Day of Civic Hacking events 
+in [2013](https://www.consumerfinance.gov/blog/we-participated-in-the-national-day-of-civic-hacking/), 
+[2014](https://www.consumerfinance.gov/blog/we-participated-in-the-national-day-of-civic-hacking-again/),
```
Comment (marteki) [inline _posts/2017-03-27-open source update.md]: At narrower screen sizes, there's a hard break after the link for the 2014 event. Seems not intentional?

screen shot 2017-05-22 at 3 51 17 pm

File: _posts/2017-03-27-open source update.md (hunk 1 above)
Comment (ielerol) [inline _posts/2017-03-27-open source update.md]: oh yeah, good point

**Comment by Scotchester** [comment]
Looks good! 👍

**Comment by marteki** [comment]
Coming back to this after two weeks. I notice some oddness around line spacing for the bulleted lists, but that is 100% out of scope for this PR.

👍  Approving and merging this!

**Pull request merged**